Techiral/A-Z-Python-Projects
# Merged I/image-processing

### What does this PR do?
This pull request adds an image processing feature to the project. It allows users to apply various operations to images, including Gaussian Blur, Brightness Adjustment, Contrast Adjustment, and Rotation.

### Changes Made
- Added an image processing script (`main.py`) to the project.
- Incorporated options for different image operations.
- Improved the project's functionality and versatility.

### How to Test
Reviewers can test the image processing functionality by following these steps:
1. Clone the repository.
2. Run the script using Python: `python main.py`.
3. Choose an image processing option from the menu and provide the required input.
4. Verify that the selected operation is applied correctly, and the modified image is saved.


Your feedback and suggestions for improvement are appreciated. Please review this pull request and consider merging it into the project.
